packages/desktop-client/src/components/reports/reports/Summary.tsx

```
@@ -219,7 +224,7 @@ function SummaryInner({ widget }: SummaryInnerProps) {
       );
       return;
     }
-
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)